### PR TITLE
[PRM-344] Make late appeal reason mandatory

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

**Comment by AndrewRButcher** [comment]
Screenshot of new error message
Screenshot 2021-06-16 at 10 25 42
Change requires update to ATs before merging

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
See #44 for revised error rendering

**Review by LewisMitchell — APPROVED** [review]

**Comment by LewisMitchell** [comment]
Waiting on AT changes

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
See ticket for changes.

**Comment by AndrewRButcher** [comment]
Screenshot 2021-06-16 at 13 36 28
Changes made based on UX feedback

File: conf/messages.cy
Comment (LewisMitchell) [inline conf/messages.cy:192]: Sorry to be a pain, but could you copy the makingALateAppeal.error.required key-value over from the English file?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: conf/messages.cy
Comment (AndrewRButcher) [reply conf/messages.cy]: Done

**Review by LewisMitchell — APPROVED** [review]
Awesome, ready to merge when AT's are in.

**Pull request merged**
